js/UcrBuildings.js

```diff
@@ -23,3 +23,8 @@ getBuildingLocation = function (bldg, room) {
 
     return "<a target=\"_blank\" href=\"" + url + "\">" + name + " " + room + "</a>";
 };
+
+getBuildingDesc = function(bldg) {
+  var name = buildingNames[bldg];
+  return name;
+}
```

Comment:
Yeah that's fine, I mostly did it since there's no private/protected variables in js and I got somewhat lazy. It's all going to get rewritten though.